roles/libvirt_manager/tasks/create_vms.yml

```diff
@@ -48,9 +45,7 @@
         chdir: "{{ _chdir }}"
 
     - name: "Ensure file ownership and rights for {{ vm }}"
-      vars:
-        _vm_img: >-
-          {{ vm }}.qcow2
+      become: true
```

Comment:
hmmm isn't that task nested under `name: "Create VM overlays or images for {{ vm_type }}"` block, meaning we have the `become: true` already?